app/views/contexts/hub_context.py

```diff
@@ -86,13 +86,15 @@ def get_context(self, survey_complete, enabled_section_ids) -> Mapping:
         }
 
     def get_row_context_for_section(
-        self, section_name: str, section_status: str, section_url: str
+        self, section_name: str, section_status: str, section_id: str, section_url: str
```

Comment:
The new argument should be added to the end of the argument list. We should also name it `row_id` rather than `section_id`, as `section_id` means something else in other places.